.github/workflows/pr-validation.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       
       - run: make build
       - name: Run docker and check its output
-        run: if docker run -t checkmarx/2ms:latest | grep "no scan plugin initialized"; then
+        run: if docker run -t checkmarx/2ms:latest confluence --url dump | grep "confluence credentials were not provided"; then
```

Comment:
you can use `-v` flag instead, which prints the version. 
The message `confluence credentials were not provided` may be changed